gemelli/factorization.py

```diff
@@ -277,6 +277,9 @@ def _fit(self):
             # combine all n_components
             subject_temp_trajectory = np.array(subject_temp_trajectory).T
             feature_temp_trajectory = np.array(feature_temp_trajectory).T
+            # double check check centered
+            subject_temp_trajectory -= subject_temp_trajectory.mean(axis=0)
+            feature_temp_trajectory -= feature_temp_trajectory.mean(axis=0)
```

Comment:
You should add unit tests to `gemelli.tests.test_factorization` to check if the trajectories are centered.

Reply:
Done -- added to the standalone ctf tests.